cpu/common/libutil.sh

```diff
@@ -17,19 +17,20 @@
 # Boston, MA 02110-1301, USA.
 #
 
-source $(dirname $(readlink -f $BASH_SOURCE))/libbkrm.sh
-source ../../cki_lib/lib.sh
+source $(dirname $(readlink -f $BASH_SOURCE))/../../cki_lib/libcki.sh
 
 TMPDIR=${TMPDIR:-"/tmp"}
 MSR_TOOLS_SRC_URL="https://github.com/intel/msr-tools.git"
 MSR_TOOLS_DST_DIR="$TMPDIR/msr-tools"
 
 function is_rhel7
 {
-    [ ! -e /etc/redhat-release ] && return 0
-    rel=$(cat /etc/redhat-release)
-    ver=$(echo "${rel##*release}" | awk '{$1=$1;print}' | cut -d '.' -f 1)
-    [ "$ver" -eq 7 ] && return 1 || return 0
+    [[ -e /etc/redhat-release ]] && return 0
+
+    typeset rel=$(cat /etc/redhat-release)
+    typeset ver=$(echo "${rel##*release}" | awk '{$1=$1;print}' | \
```

Comment:
I know this was there from before, but: what does `$1=$1` do in awk?

Reply:
Well, it looks weird. I updated it via sed + awk.
```patch
-    typeset rel=$(cat /etc/redhat-release)
-    typeset ver=$(echo "${rel##*release}" | awk '{$1=$1;print}' | \
-                  cut -d '.' -f 1)
+    typeset ver=$(sed 's/.*release//' /etc/redhat-release | \
+                  awk '{print $1}' | \
+                  awk -F'.' '{print $1}')
```
BTW, the patch is tested on Fedora, RHEL8 and RHEL7 manually. For test log, please refer to:
* http://pastebin.test.redhat.com/798166